packages/java/endpoint/src/main/java/dev/hilla/crud/CrudService.java

```diff
@@ -26,4 +26,25 @@ public interface CrudService<T> {
      */
     List<T> list(Pageable pageable, @Nullable Filter filter);
 
+    /**
+     * Updates (or creates) the given object and returns a potentially updated
+     * object.
+     * <p>
+     * If you store the object in a SQL database, the returned object might have
+     * a new id or updated consistency version.
+     *
+     * @param value
+     *            the object to update/store
+     * @return the fresh object or {@code null} if no object was found to update
+     */
+    @Nullable
+    T update(T value);
```

Comment:
How about using `save` here? Update makes it look like there should be a `create`, and you need to check the API docs to verify that this can also create items. `Save` is already familiar terminology from JPA.

Reply:
That would maybe fit better, I will change it

Reply:
Done